code/modules/mining/mine_turfs.dm

```diff
@@ -26,15 +26,18 @@
 	var/datum/artifact_find/artifact_find
 	var/scan_state = null //Holder for the image we display when we're pinged by a mining scanner
 	var/busy = 0 //Used for a bunch of do_after actions, because we can walk into the rock to trigger them
+	var/mined_type = /turf/unsimulated/floor/asteroid //Holds the turf that appears when the mineral is destroyed
 
-	var/mined_type = /turf/unsimulated/floor/asteroid
-
-/turf/unsimulated/mineral/air
+turf/unsimulated/mineral/air
 	oxygen = MOLES_O2STANDARD
 	nitrogen = MOLES_N2STANDARD
 	temperature = T20C
 	mined_type = /turf/unsimulated/floor/asteroid/air
 
+/turf/unsimulated/mineral/snow
+	//icon = 'icons/turf/snowwalls.dmi'
+	mined_type = /turf/snow/undersnow
```

Comment:
Gotta change all instances of /turf/snow/undersnow to /turf/snow/permafrost or whatever the dug variant is